requests/utils.py

```diff
@@ -42,11 +48,11 @@
 if platform.system() == 'Windows':
     # provide a proxy_bypass version on Windows without DNS lookups
 
+    if is_py3:
```

Comment:
I confess to not knowhing why `winreg` or `netrc` were imported inside functions but I suspect we've had issues in the past. Delaying imports until we have determined them to be necessary is a *valid* use-case for not globally importing modules.